crates/subspace-networking/Cargo.toml

```diff
@@ -51,11 +56,12 @@ features = [
     "tcp-tokio",
     "websocket",
     "yamux",
+    "serde"
 ]
 
 [dev-dependencies]
 anyhow = "1.0.57"
 clap = { version = "3.1.18", features = ["color", "derive"] }
 env_logger = "0.9.0"
 rand = "0.7.2"
-tracing-subscriber = "0.2"
+tracing-subscriber = "0.2"
```

Comment:
Please configure your IDE to insert new lines automatically, I already asked about this before.

Reply:
:-( Changed my IDE recently.Sorry.